librad/src/git.rs

```diff
@@ -149,16 +152,14 @@ impl GitProject {
         let mut pgp_key = key.clone().into_pgp(&nickname, fullname)?;
 
         // Link all metadata to the tip of the default branch
-        let default_branch = metadata.default_branch.clone();
+        let default_branch = metadata.default_branch().to_owned();
```

Comment:
Yes, but not too important.

I always use `to_owned` to say "that is what I wanted but I need to own it".
I reserve `clone` to "I am duplicating this thing because I will mutate the result to turn it into something else", or similar uses.

But this is just personal style and I can adjust to different styles.

Reply:
I haven't grown an opinion about it yet :sweat_smile: Thought you could shed some light because the documentation of `ToOwned` just says it's "more general than clone", which makes no sense to me.